package/MDAnalysis/coordinates/base.py

```diff
@@ -107,6 +107,12 @@
    .. autoattribute:: dimensions
    .. autoattribute:: triclinic_dimensions
    .. autoattribute:: volume
+   .. attribute:: data
+
+      :class:`dict` that holds arbitrary per Timestep data
+
+      .. versionadded:: 0.11.0
```

Comment:
2.0?

Reply:
This is a retrospective addition of docs for `data`, which would match release 0.11.0.